tests/test_sklearn_one_vs_rest_classifier_converter.py

```diff
@@ -363,7 +364,8 @@ def test_ovr_classification_int_ensemble(self):
     @ignore_warnings(category=warnings_to_skip)
     def test_ovr_classification_float_binary_ensemble(self):
         model, X = fit_classification_model(
-            OneVsRestClassifier(GradientBoostingClassifier()), 2)
+            OneVsRestClassifier(
+                GradientBoostingClassifier(loss="deviance")), 2)
```

Comment:
why do you need to change that file?

Reply:
try to resolve the CI pipeline failed case, but seems no necesary. I will reverse the change.